src/Script/Script.ts

```diff
@@ -17,6 +17,7 @@ export class Script implements ContentFile {
   // Ram calculation, only exists after first poll of ram cost after updating
   ramUsage: number | null = null;
   ramUsageEntries: RamUsageEntry[] = [];
+  ramCalculationError?: string | null = null;
```

Comment:
This shouldn't be both optional *and* nullable. Nullable is preferred, for monomorphism.

Reply:
forgot to remove that